### PR TITLE
DO-299: Added support to make email, phone & website clickable.

### PR DESCRIPTION
**Pull request by dhruvlahoti**

### CODE REVIEW

File: src/components/common/CardView/CardInfo/CardInfo.jsx [hunk 1]
```diff
@@ -33,14 +33,14 @@ const CardInfo = (props) => {
...
-            {removeLastString({gender, age})}
+            {removeLastString({ gender, age })}
           </div>
         </div>
       </div>
       <div className="cardInfo-container__details">
         <div className="cardInfo-container__details-left">
-          <div className="cardInfo-container-phone"><IconPhone />{primaryNumber || 'NA'}</div>
-          <div className="cardInfo-container-email"><IconMail />{email || 'NA' }</div>
+          <div className="cardInfo-container-phone"><IconPhone />{<a href={`tel:${primaryNumber}`}>{primaryNumber}</a> || 'NA'}</div>
```
Comment (mohitsaladiya) [inline src/components/common/CardView/CardInfo/CardInfo.jsx:42]: Use Hyperlink component here and in other files as well.
```suggestion
          <div className="cardInfo-container-phone"><IconPhone />{<Hyperlink href={`tel:${primaryNumber}`}>{primaryNumber}</a> || 'NA'}</div>
```

Comment (dhruvlahoti) [reply src/components/common/CardView/CardInfo/CardInfo.jsx]: Done

File: src/utils/common/formatters/formmatters.js [hunk 2]
```diff
@@ -72,30 +73,30 @@ export const removeLastStringSmallAddress = (item) => {
...
 export const removeLastStringContactDetails = (item) => {
-  let contactDetails = item.primaryNumber ? `${item.primaryNumber}/` : '';
+  let contactDetails = item.primaryNumber ? <a href={`tel:${item.primaryNumber}`}>{item.primaryNumber}</a> : '';
   if (item.alternateNumber) {
-    contactDetails = `${contactDetails} ${item.alternateNumber}`;
-  }
-  if (contactDetails.charAt((contactDetails.length) - 1) === '/') {
-    contactDetails = contactDetails.substring(0, (contactDetails.length) - 1);
+    return (
+      <div style={{ marginLeft: '0px' }}>
```
Comment (mohitsaladiya) [inline src/utils/common/formatters/formmatters.js:86]: We can not return component from utils. 
Utilize the method response in component and render jsx code on the basis of that only.

Comment (dhruvlahoti) [reply src/utils/common/formatters/formmatters.js]: Done

**Pull request merged**